src/apps/Hosting/HostingDialogDomainTable.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import { Table, TableHeader, TableRow, TableHeaderColumn, TableBody } from 'material-ui';
+
+import HostingDialogDomainRow from './HostingDialogDomainRow';
+import HostingDialogDomainsList from './HostingDialogDomainsList';
+
+const HostingDialogDomainTable = ({
+  handleAddNewDomain,
+  handleRemoveDomain,
+  handleChangeDomains,
+  domains,
+  handleChangeNewDomain,
+  newDomain
+}) => {
+  const styles = {
+    tableRow: {
+      display: 'flex'
+    },
+    tableHeaderColumn: {
+      display: 'flex',
+      alignItems: 'center'
+    },
+    tableHeaderColumnButton: {
+      justifyContent: 'center'
+    },
+    tableBody: {
+      display: 'block',
+      maxHeight: 240
+    }
+  };
+
+  return (
+    <div>
+      <Table>
+        <TableHeader
```

Comment:
I would change this to a separate method like: `renderTableHeader`, `renderTableBody` etc.